README.md

```diff
@@ -147,8 +147,40 @@ sudo dpkg -i postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
 sudo apt-get install postgresql-9.4
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.4-tsearch-extras_0.1_amd64.deb
 sudo dpkg -i postgresql-9.4-tsearch-extras_0.1_amd64.deb
+```
+
+On Fedora 22:
 
-# Then, all versions:
+```
+sudo dnf install libffi-devel memcached rabbitmq-server openldap-devel python-devel redis postgresql-server postgresql-devel postgresql libmemcached-devel
+wget https://launchpad.net/~tabbott/+archive/ubuntu/zulip/+files/tsearch-extras_0.1.3.tar.gz
+tar xvzf tsearch-extras_0.1.3.tar.gz
+cd ts2
+make
+sudo make install
+
+# now create the missing dictionary files
+sudo touch /usr/share/pgsql/tsearch_data/english.stop
+sudo touch /usr/share/pgsql/tsearch_data/en_us.dict
+sudo touch /usr/share/pgsql/tsearch_data/en_us.affix
+
+# Edit the postgres settings:
+sudo vi /var/lib/pgsql/data/pg_hba.conf
+
+# Add these two lines after line 80:
+local   zulip           zulip                                   md5
+host    all             all             127.0.0.1/32            md5
```

Comment:
We probably only need one of these two lines -- can you check which one?

Also maybe add something more robust to someone trying this on Fedora 23 later than "after line 80" ?  could be just "Above the first uncommented line"